astro/src/content/quickstarts/quickstart-php-laravel-web.mdx

```diff
@@ -0,0 +1,349 @@
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/database/migrations/2023_09_27_142009_add_fusionauth_fields_user_table.php"}
+  lang="php" />
+
+Run the command below to update your MariaDB database with the new User definition.
+
+```shell
+docker compose exec lara_app php artisan migrate
+```
+
+## Customization
+
+Now that authentication is done, the last task is to create example pages that a user can access only when logged in.
+
+### CSS And HTML
+
+Copy over some CSS and an image from the example app.
+
+```shell
+cp ./complete-application/laravel/public/changebank.css ./your-application/public/changebank.css
+cp ./complete-application/laravel/public/money.jpg ./your-application/public/money.jpg
+```
+
+Next, create an `index.blade.php` file.
+
+```shell
+touch your-application/resources/views/index.blade.php
+```
+
+Paste the following code into the file.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/resources/views/index.blade.php"}
+  lang="php" />
+
+The index page contains nothing to note except a link to the login page `<a href="/login">`.
+
+Next, create an `account.blade.php` file.
+
+```shell
+touch your-application/resources/views/account.blade.php
+```
+
+Paste the following code into the file.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/resources/views/account.blade.php"}
+  lang="php" />
+
+The account page displays the user's email from FusionAuth, as you'll see later when you create the route `<p class="header-email">{{$email}}</p>`.
+
+Next, create a `change.blade.php` file.
+
+```shell
+touch your-application/resources/views/change.blade.php
+```
+
+Paste the following code into the file.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/resources/views/change.blade.php"}
+  lang="php" />
+
+The change page uses some Blade conditional statements to display different divs depending on the state returned by the form POST `@if ($state['error'] && $state['hasChange'])`.
+
+The change page also includes a `@csrf` anti-forgery token in the form at the bottom to prevent {frontmatter.technology} throwing page expired errors.
+
+### Route
+
+In the file `your-application/routes/web.php`, overwrite everything with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/routes/web.php"}
+  lang="php" />
+
+The homepage route `/` checks whether you are logged in and redirects you to your account page if you are.
+
+The login route replaces the Docker service name `fusionauth` with `localhost` in `$url = str_replace('fusionauth', 'localhost', $url);`. The former is used in the {frontmatter.technology} service to access FusionAuth, while the latter needs to be displayed in your browser URL bar, which doesn't know about Docker since it's on your physical machine. In future apps, if you aren't running in Docker, you can use the first commented line instead of this replacement.
```

Comment:
can we add a comment in that PHP file that says "if you aren't running in docker, uncomment this line", please?

Reply:
changed the implementation not to use Docker so I removed this